docs/CToSwiftNameTranslation.md

```diff
@@ -212,6 +212,244 @@ Additionally, typedefs for `void *` or `const void *` that are themselves annota
 If a typedef's underlying type is itself a "CF pointer" typedef, the "alias" typedef will be imported as a regular typealias, with the suffix "Ref" still dropped from its name (if present) unless doing so would conflict with another declaration in the same module as the typedef.
 
 
+## Objective-C Methods
+
+Objective-C methods are classified into one of five categories:
+
+1. Instance methods that are part of the `init` method family whose selector's first word is "init" are considered "normal" initializers.
+
+2. Class methods that return the type of the containing class or `instancetype` are considered "factory initializers" if a "leading type match" succeeds against the selector, using the containing type (see [CToSwiftNameTranslation-OmitNeedlessWords.md][]).
+
+    _As an exception, no-argument factory methods cannot have any additional text *after* the matching portion._
+
+3. Accessor methods are associated with a property declaration (`@property`).
+
+4. Subscript methods have one of Objective-C's special subscript selectors (`objectAtIndexedSubscript:`, `objectForKeyedSubscript:`, `setObject:atIndexedSubscript:`, or `setObject:forKeyedSubscript:`).
+
+5. Other methods are imported as plain methods.
+
+If a method overrides a superclass or adopted protocol method, the Swift name of the overridden method will be used for consistency. If there's more than one such name, one is chosen arbitrarily.
```

Comment:
Could you give a small example of how the case of "more than one such name" would arise? Can a single Obj-C method override multiple methods at once?

Reply:
I guess I can put an example in. It's mostly just `NS_SWIFT_NAME` on one protocol requirement and not on another requirement in a different protocol with the same selector. It's such a terrible situation that I don't want to spend much time on it. :-(

Reply:
I just asked for my own curiosity. I didn't realize you could override multiple methods at once. You don't need to put it in if you think it doesn't warrant discussion.

Reply:
"override" is a fuzzy term here; it's being used for satisfying protocol requirements too. Maybe that's what should be clarified.

Reply:
(Clang uses this terminology, but normal Objective-C programmers don't.)